admin_app/src/app/content/page.tsx

```diff
@@ -97,48 +112,37 @@ const CardsPage = () => {
           lg={12 - sidebarGridWidth + 1}
           sx={{
             display: openSidebar ? { xs: "none", sm: "none", md: "block" } : "block",
+            paddingInline: 2,
           }}
         >
+          <Typography
+            sx={{ paddingTop: 6, paddingLeft: 3 }}
+            variant="h4"
+            align="left"
+            color="primary"
+          >
+            Manage Contents
+          </Typography>
+          <Typography
+            sx={{ paddingTop: 2, paddingLeft: 3 }}
+            variant="body1"
+            align="left"
+          >
+            On this page, you can view, edit, delete, and test contents.
+          </Typography>
           <Layout.FlexBox
             flexGrow={1}
             alignItems="center"
             gap={sizes.baseGap}
-            paddingTop={8}
+            paddingTop={5}
           >
-            <Layout.FlexBox
-              gap={sizes.smallGap}
-              sx={{
-                width: "70%",
-                maxWidth: "500px",
-                minWidth: "200px",
-              }}
-            >
-              <SearchBar searchTerm={searchTerm} setSearchTerm={setSearchTerm} />
-              <Layout.FlexBox
-                alignItems="center"
-                sx={{ flexDirection: "row", justifyContent: "center" }}
-                gap={sizes.smallGap}
-              >
-                <Autocomplete
-                  multiple
-                  limitTags={3}
-                  id="tags-autocomplete"
-                  options={tags}
-                  getOptionLabel={(option) => option.tag_name}
-                  noOptionsText="No tags found"
-                  value={filterTags}
-                  onChange={(event, updatedTags) => {
-                    setFilterTags(updatedTags);
-                  }}
-                  renderInput={(params) => (
-                    <TextField {...params} variant="standard" label="Filter by tags" />
-                  )}
-                  sx={{ width: "80%", color: appColors.white }}
-                />
-              </Layout.FlexBox>
-            </Layout.FlexBox>
             <CardsUtilityStrip
```

Comment:
I don't know how I feel about the left aligned design. But happy to go with it if everyone thinks it' better.

Reply:
Yeah decided this with Anh and Tanmay on a design call so will leave it for now if that's okay. It's the cleanest/most comprehensible grouping that allows for a page title and description I think.